examples/ts_project_indirect_npm/BUILD.bazel

```diff
@@ -0,0 +1,25 @@
+load("@aspect_rules_ts//ts:defs.bzl", "ts_project")
+
+# Importing js_lib2 via linked npm package
+ts_project(
+    name = "named-import",
+    srcs = ["import-named.ts"],
+    tsconfig = {},
+    deps = ["//examples:node_modules/@myorg/js_lib2"],
+)
+
+# Importing js_lib2 via relative path
+ts_project(
+    name = "relative-import",
+    srcs = ["import-relative.ts"],
+    tsconfig = {},
+    deps = ["//examples/js_lib2:js_library"],
+)
+
+# Importing js_lib via linked npm package without direct dependency on it
+ts_project(
+    name = "indirect-import",
+    srcs = ["import-transitive.ts"],
```

Comment:
This should fail because it doesn't directly depend on `//examples/js_lib`. Is this a known issue?